chia/server/node_discovery.py

```diff
@@ -284,7 +314,9 @@ async def _connect_to_peers(self, random) -> None:
                         addr = None
                         continue
                     # only consider very recently tried nodes after 30 failed attempts
-                    if now - info.last_try < 600 and tries < 30:
+                    # attempt a node once per 2 hours if we lack connections to increase the chance
+                    # to try all the peer table.
+                    if now - info.last_try < 2 * 3600 and tries < 30:
```

Comment:
somewhat unrelated: is `now` posix time as UTC? It's important it doesn't have daylight savings for instance, we need it to be monotonic.